red/df1.js

```diff
@@ -126,11 +126,15 @@ module.exports = function (RED) {
         this.setMaxListeners(0);
 
         this.getDf1Session = () => {
-            const df1protocol = df1.df1protocol();            
+            const df1protocol = df1.df1Protocol;            
 
-            if(df1protocol) return df1protocol.dataLinkSession();
+            if(df1protocol) return df1protocol.dataLinkSession;
         };
 
+        this.df1Endpoint = () => {
+            return df1;
+        }
+

```

Comment:
I think it would be better to avoid using two different functions here for accessing the same property. If you're already returning the `df1` variable, why not directly access it to get the `dataLinkSession`?